Misc/NEWS.d/next/Documentation/2024-04-17-22-16-19.gh-issue-116932.lETyxa.rst

```diff
@@ -0,0 +1 @@
+Add notes on how to report python documentation theme bugs
```

Comment:
News entry is [not needed](https://devguide.python.org/core-developers/committing/#changes-that-require-news-entries).